pkg/test/resourcefixture/testdata/basic/gkebackup/v1alpha1/gkebackupbackup/gkebackupbackup-minimal/_http.log

```diff
@@ -470,26 +470,28 @@ X-Xss-Protection: 0
   },
   "autopilot": {},
   "autoscaling": {
-    "autoprovisioningNodePoolDefaults": {
```

Comment:
Any idea why this is removed?

Reply:
Because this is an incorrect behavior in the mock API.